virt/virt-2-4-release-notes.adoc

```diff
@@ -8,56 +8,105 @@ toc::[]
 
 include::modules/virt-what-you-can-do-with-virt.adoc[leveloffset=+2]
 
-=== {VirtProductName} support
+[id="virt-2-4-new"]
+== New and changed features
+. Placeholder for new content.
 
-:FeatureName: {VirtProductName}
-include::modules/technology-preview.adoc[leveloffset=+2]
+//CNV-4686 - Container-native Virtualization x" in the SVVP catalog listing
 
-== New and changed features
+//CNV-4687 - Ensure platform passes Windows Server Virtualization Program with RHCOS workers.
+
+//CNV-5371 - Import existing virtual machines from Red Hat Virtualization to OpenShift Virtualization.
+
+[id="virt-2-4-networking-new"]
+=== Networking
+. Placeholder for new content.
+
+#CNV-5286 - SR-IOV Networking to provide a basic fast datapath option
```

Comment:
as above: Not a big deal but this would be better as a comment
(for next line as well)

Reply:
Changed the text to a comment with //.

Reply:
Added // to comment this text.